impeller/display_list/dl_unittests.cc

```diff
@@ -46,19 +46,6 @@ flutter::DlColor toColor(const float* components) {
 using DisplayListTest = DlPlayground;
 INSTANTIATE_PLAYGROUND_SUITE(DisplayListTest);
 
-TEST_P(DisplayListTest, DrawPictureWithAClip) {
```

Comment:
Even though the name of this test was "DrawPictureWithAClip", it was actually drawing a DisplayList with a clip - should it be eliminated here?